From be56949d67d9032f391f95ff2c56923d3da015eb Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 23 Nov 2018 20:24:05 +0900
Subject: [PATCH 527/909] arm64: dts: r8a7796: Add cpg reset for LVDS Interface

commit 7b95e0e4c8c1114132c4d7d32f7e458973ebe734 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

It is necessary to reset the LVDS Interface according to display on/off.
Therefore, this patch adds CPG reset properties in DU device node
for the R8A7796 SoC.

Reported-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
[takeshi.kihara.df: rebased]
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index 75e71d7a819d..9844d02cc607 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -2779,6 +2779,10 @@
 				 <&cpg CPG_MOD 723>,
 				 <&cpg CPG_MOD 722>;
 			clock-names = "du.0", "du.1", "du.2";
+			resets = <&cpg 724>,
+				 <&cpg 724>,
+				 <&cpg 722>;
+			reset-names = "du.0", "du.1", "du.2";
 			status = "disabled";
 
 			vsps = <&vspd0 &vspd1 &vspd2>;
-- 
2.17.1

